Tests/SonarQube.MSBuild.Tasks.IntegrationTests/Infrastructure/BuildAssertions.cs

```diff
@@ -67,7 +69,43 @@ public static void AssertPropertyDoesNotExist(ProjectInstance projectInstance, s
 
             Assert.IsNull(propertyInstance, "Not expecting the property to exist. Property: {0}, Value: {1}", propertyName, value);
         }
-        
+
+        /// <summary>
+        /// Checks whether there is a single "SonarQubeSetting" item with the expected name and setting value
+        /// </summary>
+        public static void AssertExpectedAnalysisSetting(BuildResult actualResult, string settingName, string expectedValue)
```

Comment:
This method was moved to a shared location. The next method ("AssertAnalysisSettingDoesNotExist") is new.